docs/user-guide/query/gremlin/gremlin-limits.md

```diff
@@ -22,14 +22,16 @@ Current known limitations or bugs:
 - Edge IDs in GafferPop are not the same as in standard Gremlin. Instead of `g.E(11)`
   edge IDs take the format `g.E("[source, dest]")` or `g.E("[source, label, dest]")`.
 - The entity group `id` is reserved for an empty group containing only the
-  vertex ID, this is currently used as a workaround for other limitations.
+  vertex ID, this is currently used as a workaround for other limitations. One such
+  use is for holding 'orphaned' vertexes, these are vertexes on an edge that do not
```

Comment:
I thought these should be vertices but Google just told me that vertexes is also a word
So can leave if we've been using vertexes elsewhere